src/components/Card.js

```diff
@@ -0,0 +1,42 @@
+import { Modal } from '@material-ui/core';
+import styled from '@emotion/styled';
+import CloseIcon from '@material-ui/icons/Close';
+
+import colors from 'styles/colors';
+
+const CloseButton = styled(CloseIcon)({
+  cursor: 'pointer',
+  position: 'absolute',
+  top: 5,
+  right: 5,
+  borderRadius: '50%',
+  '&:hover': {
+    backgroundColor: colors.lightGray,
+  },
+});
+
+export default function Card({ children, isOpen, close }) {
+  return (
+    <Modal
```

Comment:
Is there a reason why you used styled for the Close Button and not the other components?

Reply:
iirc I was just adding styling inline and then used styled for that component so I could leverage the hover state. I went ahead and made changes to make this consistent.